InfiniTAM/infinitam_ros_node.cpp

```diff
@@ -317,26 +533,23 @@ void InfinitamNode::SetUpSources() {
     printf("using rgb images: %s\nusing depth images: %s\n",
            depth_image_filename, rgb_image_filename);
     if (filename_imu == NULL) {
-      image_source_ = new ImageFileReader(calibration_filename,
-                                          depth_image_filename,
-                                          rgb_image_filename);
+      image_source_ = new ImageFileReader(
+          calibration_filename, depth_image_filename, rgb_image_filename);
     } else {
       printf("using imu data: %s\n", filename_imu);
-      image_source_ = new RawFileReader(calibration_filename,
-                                        depth_image_filename,
-                                        rgb_image_filename, Vector2i(320, 240),
-                                        0.5f);
+      image_source_ =
+          new RawFileReader(calibration_filename, depth_image_filename,
+                            rgb_image_filename, Vector2i(320, 240), 0.5f);
```

Comment:
Why do we have this set to 320, 240, but I guess this doesn't really matter for now.